src/test/java/com/google/sps/DeleteUserServletTest.java

```diff
@@ -0,0 +1,105 @@
+package com.google.sps;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.users.User;
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
+import com.google.appengine.tools.development.testing.LocalDatastoreServiceTestConfig;
+import com.google.appengine.tools.development.testing.LocalServiceTestHelper;
+import com.google.appengine.tools.development.testing.LocalUserServiceTestConfig;
+import com.google.gson.Gson;
+import com.google.sps.data.LoggedOutException;
+import com.google.sps.data.Player;
+import com.google.sps.data.PlayerDatabase;
+import com.google.sps.servlets.DeleteUserServlet;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+
+/** Tests that the DeleteUserServlet servlet correctly deletes a player from the database */
+@RunWith(JUnit4.class)
+public final class DeleteUserServletTest {
+  private static final String AUTH_DOMAIN = "email.com";
+  private static final String CURR_USER_ID = "testid";
+  private static final String USER_ID_KEY_PATH =
+      "com.google.appengine.api.users.UserService.user_id_key";
+  private static Map<String, Object> USER_ID_CONFIG =
+      new HashMap<String, Object>() {
+        {
+          put(USER_ID_KEY_PATH, CURR_USER_ID);
+        }
+      };
+  // creating a test user service
+  private final LocalUserServiceTestConfig localUserServiceTestConfig =
+      new LocalUserServiceTestConfig().setOAuthUserId(CURR_USER_ID);
+  private final LocalDatastoreServiceTestConfig localDatastoreServiceTestConfig =
+      new LocalDatastoreServiceTestConfig();
+  private final LocalServiceTestHelper helper =
+      new LocalServiceTestHelper(localUserServiceTestConfig, localDatastoreServiceTestConfig)
+          .setEnvIsAdmin(false)
```

Comment:
Should the user environment be set to admin?

Reply:
No, I don't think it's necessary here.